README.md

```diff
@@ -1,13 +1,14 @@
 # Foreman Pulp Plugin
 
-Foreman project plugin for Pulp allowing Katello hosts to interact with Pulp application services for content management
-While this plugin is part of the Foreman project, it can only be used with Katello as Foreman is not content aware without the Katello plugin.
+Foreman project plugin for Pulp allowing Katello hosts to interact with Pulp application services for content management. While this plugin is part of the Foreman project, it can only be used with Katello as Foreman is not content aware without the Katello plugin.
 
 ## Getting Started
 
 The Foreman project provides documentation on this specific plugin installation from multiple methods, distribution specific RPM, distribution specific DEB and direct manual build from source.
-The plugin can also be installed as part of the foreman-installer puppet class with the "--[no-]enable-foreman-proxy-plugin-pulp" option.
-Documentation can be found here for how to install a stand alone smart proxy https://theforeman.org/manuals/latest/index.html#4.3.1SmartProxyInstallation
+
+The plugin can also be installed as part of the foreman-installer puppet class with the `--[no-]enable-foreman-proxy-plugin-pulp` option.
```

Comment:
This is not true anymore, we've removed this and now wrap it in a different class. I'd leave it out.